cocos/2d/CCParticleSystem.cpp

```diff
@@ -588,6 +588,8 @@ ParticleSystem::~ParticleSystem()
 
 void ParticleSystem::addParticles(int count)
 {
+    if (_paused)
```

Comment:
Why `_pause` wasn't initialized to false? It will be a random value (true / false).